pkgs/applications/emulators/ryujinx/updater.sh

```diff
@@ -1,33 +1,60 @@
 #! /usr/bin/env nix-shell
-#! nix-shell -i bash -p coreutils gnused curl common-updater-scripts nuget-to-nix nix-prefetch-git jq dotnet-sdk_6
+#! nix-shell -I nixpkgs=./. -i bash -p coreutils gnused curl common-updater-scripts nuget-to-nix nix-prefetch-git jq dotnet-sdk_6
```

Comment:
I don't think running this from the nixpkgs root will work, since the script writes to paths relative to the packages directory (`deps.nix`) and we read the old version from `./default.nix`. Maybe we could `pushd` into it?

Reply:
Ah, makes sense :+1: